test/app-tap/tarantoolctl.test.lua

```diff
@@ -133,6 +133,26 @@ local function tctl_command(dir, cmd, args, name)
     return run_command(dir, command)
 end
 
+local function tctl_wait_eval(dir, app, attempts)
```

Comment:
Please write arguments help instructions before the function (too hard to read after w/o it).

Reply:
Added.